<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>McpMainWindow</class>
 <widget class="QMainWindow" name="McpMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCP Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="mqttServerLabel">
      <property name="text">
       <string>MQTT Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="mqttServerEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="clientIdLabel">
      <property name="text">
       <string>Client ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="clientIdEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="sendMessageGroup">
      <property name="title">
       <string>Send Message</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="recepientLabel">
         <property name="text">
          <string>Recepient</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="recepientEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="messageLabel">
         <property name="text">
          <string>Message</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="messageEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="logMessagesEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>525</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="sendMessageButton">
      <property name="text">
       <string>Send Message</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
